python/cudf_polars/tests/test_groupby.py

```diff
@@ -258,7 +258,8 @@ def test_groupby_maintain_order_random(nrows, nkeys, with_nulls):
             )
         )
     q = df.lazy().group_by(key_names, maintain_order=True).agg(pl.col("value").sum())
-    assert_gpu_result_equal(q)
+    # The streaming executor is too slow for large n_rows with blocksize_mode="small"
```

Comment:
I don't know if there's a better way to find slow tests, other than watching out for tests that large number of rows and the default engine / `blocksize_mode`. pytest does have a `--durations` flag we could include.

Reply:
Are you saying we could use `--durations` to find the slow tests and then possibly set `blocksize="default"` on those tests?